packages/rke2-whereabouts/charts/values.yaml

```diff
@@ -6,7 +6,7 @@ image:
   repository: rancher/hardened-whereabouts
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: v0.8.0-build20241011
+  tag: v0.8.0-build20240910
```

Comment:
revert https://github.com/rancher/rke2-charts/pull/546 ?

Reply:
ah! good catch Mike. Our automation is faulty

Reply:
Fixed by https://github.com/rancher/rke2-charts/pull/495